user-guide/Advanced_Functionality/Databases/Local_database_on_WSL/Decommissioning_WSL.md

```diff
@@ -0,0 +1,18 @@
+---
+uid: Decommissioning_WSL
+---
+
+# Decommissioning WSL
+
+If you no longer need to run Cassandra Cluster and OpenSearch from WSL because you migrated to [Storage as a Service (STaaS)](xref:STaaS) or [configured dedicated clustered storage](xref:Configuring_dedicated_clustered_storage), it is recommended to stop WSL so it no longer consumes any resources.
+
+Follow the below steps to decommission WSL:
+
+1. In an elevated Powershell window, run the following command:
+
+    ```powershell
+    wsl --shutdown
```

Comment:
Also add: `wsl  --unregister Ubuntu-22.04`

This will delete the distribution of the disk aswell.

If you made changes to your distribution and want to keep it you can export it with `wsl --export`

You can always re-import with `wsl --import`. (Either your own exported  distro or the one at *C:\Skyline Communications\WSLBackup*.)